internal/kafka/internal/services/data_plane_cluster.go

```diff
@@ -432,7 +432,9 @@ func (d *dataPlaneClusterService) scaleUpThresholds(status *dbapi.DataPlaneClust
 // to consider that a kafka cluster has capacity available
 func (d *dataPlaneClusterService) minimumKafkaCapacity() *dataPlaneComputeNodesKafkaCapacityAttributes {
 	return &dataPlaneComputeNodesKafkaCapacityAttributes{
-		Connections: d.KafkaConfig.KafkaCapacity.TotalMaxConnections,
-		Partitions:  d.KafkaConfig.KafkaCapacity.MaxPartitions,
+		// using hardcoded values because the dynamic scaling implementation will be revised in the future and
```

Comment:
I'd suggest to prefix these kind of comments with `FIXME` or `TODO` so that it will be easy to find them
```suggestion
		// FIXME: using hardcoded values because the dynamic scaling implementation will be revised in the future and
```

Reply:
I personally think a TODO or FIXME is always good, anyone who stumbles across such prefixes can kick start a conversation if we had missed anything.

Reply:
The thing here is that, dynamic scaling implementation as it is implemented now will be gone. It'll be a complete rewrite . 
This part of the code is not used by anyone at the moment, other than the integration tests. Someone could just remove the code and it'll be the same. 

> .. anyone who stumbles across such prefixes can kick start a conversation if we had missed anything.

If we have JIRAs/issues for the situation at handy for it, then nothing is missed.

To be clear, I agree on having TODOs comment with associated JIRAs for something that needs improving. But adding them to something that we know for sure is going to be re-written (whole of it) in a couple of sprint time (issue to track this are there) isn't really worth it.

Reply:
If we create JIRAs then and there no issues, often we miss that (or like we discussed, when the fix is simple compared to time to create JIRA, we dont create them) , thats is where TODOs come in handy :) .

Reply:
It is a new comment added in this PR. Since there is no harm in having the TODO/FIXME prefix, I'd prefer to have it anyway.

Reply:
I agree with Manyanda, there not really anything to do or to fix about this. In the future it will be removed and replaced by something different. Adding  `TODO` or `FIXME` to those lines would suggest that they have to be updated which is not the case.

Reply:
That is exactly what the comment is about: what do we have TO DO? Remove it in the future. So it is a TODO. The rest of the comment explains when it has to be done and how. 

However, I will agree with the majority here, so I'm fine with not adding it.